### PR TITLE
Cnnetantwo 57 3

### PR DESCRIPTION
**Pull request by dsover**
This branch sets some new standards for file naming conventions, and creates a content details page with a headline and save/ cancel function

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -16,8 +16,9 @@
 
 let config = {
     brainApi: {
-        uri: 'http://cnn-opp-brain-api.ref.services.ec2.dmtio.net/'
-    }
+        uri: 'http://cnn-opp-brain-api.ref.services.ec2.dmtio.net'
+    },
+    defaultAPITimeout_MS: 5000
```
Comment (jamsyoung) [inline config.js]: ```1000 * 5```

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
         "babel-resolver": "1.1.0",
         "cnn-server": "git+https://github.com/cnnlabs/cnn-server.git#0.1.0",
         "jsonp": "0.2.1",
+        "lodash.clonedeep": "^4.5.0",
```
Comment (jamsyoung) [inline package.json]: pin to an exact version

File: src/modules/content-list/__tests__/__snapshots__/content-list.spec.js.snap [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (jamsyoung) [inline src/modules/content-list/__tests__/__snapshots__/content-list.spec.js.snap]: is this file generated? or did you need to type it in?

If it is generated - is there a way to have it generated as minimized (so humans know not to try to edit it)?

If it is typed in, add a `[*.snap]` section to the `.editorconfig` file to set the indentation to 2 paces (not the default 4)

File: src/modules/headline/styles.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.container {
+    display: flex;
+}
+
+.inputElement {
+    background-color: white;
```
Comment (jamsyoung) [inline src/modules/headline/styles.css]: use hex code vs. named color (they may diff across devices (IE's white is not Firefox's white for example)

File: src/modules/headline/styles.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    border:none;
+    width: 100%;
+}
+
+.inputElementError {
+    background-color: white;
+    border:none;
```
Comment (jamsyoung) [inline src/modules/headline/styles.css]: My OCD wants a space after the colon, however the linter isn't complaining so ... its up to you.

(same for other sans-space colons)

File: src/redux/reducers/content_detail_reducer.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+            headlineChanged: false,
+            isHeadlineTextInvalid: false
+        }
+
+    case types.SAVE_CONTENT_DETAIL_PENDING:
+        return {
+            ...state,
+            fetching: true
+        }
+    case types.SAVE_CONTENT_DETAIL_SUCCESS:
```
Comment (jamsyoung) [inline src/redux/reducers/content_detail_reducer.js]: OCD - blank line above case

File: package.json (hunk 2 above)
Comment (britneydossett) [inline package.json]: Do we want a permanent version on this?

File: src/modules/content-list/__tests__/__snapshots__/content-list.spec.js.snap (hunk 3 above)
Comment (dsover) [inline src/modules/content-list/__tests__/__snapshots__/content-list.spec.js.snap]: It's auto-generated, we should check with matt and josh as they will be defining the base config for testing

File: src/redux/reducers/content_detail_reducer.js (hunk 6 above)
Comment (dsover) [inline src/redux/reducers/content_detail_reducer.js]: done

File: src/modules/headline/styles.css (hunk 5 above)
Comment (dsover) [inline src/modules/headline/styles.css]: done

File: src/modules/headline/styles.css (hunk 4 above)
Comment (dsover) [inline src/modules/headline/styles.css]: done

File: package.json (hunk 2 above)
Comment (dsover) [inline package.json]: done

Comment (dsover) [inline package.json]: done

**Pull request merged**